testdata/bom_glif.glif

```diff
@@ -0,0 +1,25 @@
+﻿<!-- this includes a byte order mark (https://en.wikipedia.org/wiki/Byte_order_mark)-->
+<?xml version="1.0" encoding="UTF-8"?>
+<glyph name=".notdef" format="2">
+  <advance width="600" height="1000"/>
+  <outline>
+    <contour>
+      <point x="145" y="51" type="line"/>
+      <point x="145" y="663" type="line"/>
+      <point x="454" y="663" type="line"/>
+      <point x="454" y="51" type="line"/>
+    </contour>
+    <contour>
+      <point x="94" y="0" type="line"/>
+      <point x="505" y="0" type="line"/>
+      <point x="505" y="714" type="line"/>
+      <point x="94" y="714" type="line"/>
+    </contour>
+  </outline>
```

Comment:
optional: guessing you don't even need an outline?

Reply:
was this intended to come with a +1?

Reply:
it was!